README.md

```diff
@@ -1,47 +1,56 @@
 # Mantle: Gluing CoreOS together
 
-This repository is a collection of utilities for the CoreOS SDK.
-
-## plume
-
-CoreOS release utility
-
-## plume index
-
-Generate and upload index.html objects to turn a Google Cloud Storage
-bucket into a publicly browsable file tree. Useful if you want something
-like Apache's directory index for your software download repository.
+This repository is a collection of utilities for the Container Linux SDK. Most of the
+tools are for uploading, running, and interacting with Container Linux instances running
+locally or in a cloud.
 
 ## kola
 
-Kola is a framework for testing software integration in CoreOS instances
+Kola is a framework for testing software integration in Container Linux instances
 across multiple platforms. It is primarily designed to operate within
-the CoreOS SDK for testing software that has landed in the OS image.
+the Container Linux SDK for testing software that has landed in the OS image.
 Ideally, all software needed for a test should be included by building
 it into the image from the SDK.
 
 Kola supports running tests on multiple platforms, currently QEMU, GCE,
-and AWS. In the future systemd-nspawn and other platforms may be added.
+AWS, VMware, and Packet. In the future systemd-nspawn and other
+platforms may be added.
 Local platforms do not rely on access to the Internet as a design
 principal of kola, minimizing external dependencies. Any network
 services required get built directly into kola itself. Machines on cloud
 platforms do not have direct access to the kola so tests may depend on
 Internet services such as discovery.etcd.io or quay.io instead.
 
 Kola outputs assorted logs and test data to `_kola_temp` for later
-inspection. This directory is completely wiped and recreated every time.
+inspection.
 
 Kola is still under heavy development and it is expected that its
 interface will continue to change. Both the CLI and test registration
 interface have upcoming changes.
 
+By default, kola uses the `qemu` platform with the most recently built image
+(assuming it is run from within the SDK).
+
 ### kola run
 The run command invokes the main kola test harness. The harness will
 run any registered tests on all platforms unless otherwise specified. It
 runs any tests whose registered names matches a glob pattern.
 
 `kola run <glob pattern>`
 
+To get a list of all tests, run
+
+`kola list`
+
+### kola spawn
+The spawn command launches Container Linux instances. Use `-s` to open a shell on
+the spawned instance.
+
+### kola mkimage
+The mkimage command creates a copy of the input image with its primary console set
```

Comment:
It's important to point out that `kola mkimage` is only for QEMU images, and specifically `coreos_*_image.bin` (rather than `coreos_*_qemu_image.img`).